### DISCUSSION TITLE
Get-PlasterTemplate : Add a possibility to retrieve all template module regardless of the version

### DISCUSSION THREAD

**Discussion by LaurentDardenne** [PowerShellOrg/Plaster#383] [Ideas]
Today _Get-PlasterTemplate_ retrieve the templates only for the current module version (1.0.1).

It not possible to known which version of the Plaster module I must use for each Template.

And to filter the templates, _Get-PlasterTemplate_ use the template module manifest not an information given by the user.

**Comment by LaurentDardenne** [comment]
I use the latest build, and the Get-Plastertemplate do not return the type of the template (Project or Item);
```Powershell
>Get-PlasterTemplate

Title        : AddPSScriptAnalyzerSettings
Author       : Plaster project
Version      : 1.0.0
Description  : Add a PowerShell Script Analyzer settings file to the root of your workspace.
Tags         : {PSScriptAnalyzer, settings}
TemplatePath : C:\Program Files\WindowsPowerShell\Modules\plaster\1.0.2\Templates\AddPSScriptAnalyzerSettings

Title        : New PowerShell Manifest Module
Author       : Plaster
Version      : 1.0.0
Description  : Creates files for a simple, non-shared PowerShell script module.
Tags         : {Module, ModuleManifest}
TemplatePath : C:\Program Files\WindowsPowerShell\Modules\plaster\1.0.2\Templates\NewPowerShellScriptModule
```

**Comment by daviwil** [comment]
I haven't finished updating the Get-PlasterTemplate cmdlet yet, Keith and I have been too busy with other projects lately to get the 1.1.0 release out with these changes.

**Comment by LaurentDardenne** [comment]
Ok thanks.
_All things come to him who waits_ :-)

**Comment by larssb** [comment]
Any update on this? Would be a great feature.

**Comment by TylerLeonhardt** [comment]
Haven't gotten around to this because of other project - feel free to contribute! Plaster is all PowerShell :)

Here's the code for `Get-PlasterTemplate`
https://github.com/PowerShell/Plaster/blob/master/src/GetPlasterTemplate.ps1
